plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -228,6 +251,64 @@ describe('CassandraDriverInstrumentation', () => {
         assertSingleSpan('cassandra-driver.execute', query.substr(0, 25));
       });
     });
+
+    describe('responseHook', () => {
+      after(() => {
+        instrumentation.setConfig({
+          responseHook: () => {},
+        });
+      });
+
+      it('adds custom attributes to span', async () => {
+        const responseAttributeName = 'response.attribute';
+        const customAttributeName = 'custom.attribute';
+        const customAttributeValue = 'custom attribute value';
+
+        instrumentation.setConfig({
+          responseHook: (span: Span, responseInfo: ResponseInfo) => {
+            const row = responseInfo.response.first();
+            span.setAttribute(responseAttributeName, row.count.toString());
```

Comment:
nit: why convert the row count to string?
Even though it's allowed, it makes more sense to me to be a number. I think the test should promote idiomatic use of attributes

Reply:
The reason for converting the count to string is because its type is Long, which can't be set as an attribute in setAttribute().
I updated the test to parse it to a number that maybe makes more sense for the count :)